packages/indexer/tests/parser_test.py

```diff
@@ -63,24 +66,9 @@ def test_parse_individual_log_event():
     }
 
 
-def test_parse_tx_logs():
+def test_parse_tx_logs(mock_log_data):
```

Comment:
erroring test

mock_log_data is some jackal data. Have 0 ideas where it is coming from

Reply:
Explain?

Reply:
mock log data isn't what it should be